core/src/main/java/hudson/WebAppMain.java

```diff
@@ -80,16 +80,19 @@
  */
 public class WebAppMain implements ServletContextListener {
 
-    // use RingBufferLogHandler class name to configure for backward compatibility
-    private static final int DEFAULT_RING_BUFFER_SIZE = SystemProperties.getInteger(RingBufferLogHandler.class.getName() + ".defaultSize", 256);
-
-    private final RingBufferLogHandler handler = new RingBufferLogHandler(DEFAULT_RING_BUFFER_SIZE) {
+    private final RingBufferLogHandler handler = new RingBufferLogHandler(RingBufferLogHandler.DEFAULT_RING_BUFFER_SIZE) {
```

Comment:
You can call the no-arg constructor because it uses `RingBufferLogHandler.DEFAULT_RING_BUFFER_SIZE`
```suggestion
    private final RingBufferLogHandler handler = new RingBufferLogHandler() {
```